src/Utils/decimal.ts

```
@@ -0,0 +1,161 @@
+
+/**
+ * Convert to number (use sparingly, only for display/charting)
+ */
+export function toNumber(value: string | number | Decimal): number {
+  return new Decimal(value).toNumber();
+}
+
+/**
+ * Convert to string (for API submission)
+ */
+export function toString(value: string | number | Decimal): string {
```

Biome (2.1.2)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)